rotkehlchen/tests/unit/decoders/test_basenames.py

```diff
@@ -0,0 +1,468 @@
+from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.tests.utils.ethereum import get_decoded_events_of_transaction
+from rotkehlchen.types import Location, Timestamp, TimestampMS, deserialize_evm_tx_hash
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.base.node_inquirer import BaseInquirer
+    from rotkehlchen.types import ChecksumEvmAddress
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('base_accounts', [['0xc37b40ABdB939635068d3c5f13E7faF686F03B65']])
+def test_basenames_register(
+        base_inquirer: 'BaseInquirer',
+        base_accounts: list['ChecksumEvmAddress'],
+) -> None:
+    tx_hash = deserialize_evm_tx_hash('0x20280b43dbcfa86cdf0703d2e9f8f2ef200839b2ee0e819d895515d3adb74eff')  # noqa: E501
+    events, decoder = get_decoded_events_of_transaction(evm_inquirer=base_inquirer, tx_hash=tx_hash)  # noqa: E501
+    timestamp, user_address, gas_amount = TimestampMS(1726738619000), base_accounts[0], '0.000001963384627852'  # noqa: E501
+    assert events == [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_amount)),
+            location_label=user_address,
+            notes=f'Burn {gas_amount} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=182,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_ETH,
+            balance=Balance(amount=ZERO),
+            location_label=user_address,
+            notes=f'Transfer base.eth node ownership of subnode yabir.base.eth to {user_address}',
+            counterparty=CPT_BASENAMES,
+            address=BASENAMES_REGISTRY,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=183,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_ETH,
+            balance=Balance(amount=ZERO),
+            location_label=user_address,
+            notes='Set Basenames address for yabir.base.eth',
+            counterparty=CPT_BASENAMES,
+            address=BASENAMES_REGISTRAR_CONTROLLER,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=186,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_ETH,
+            balance=Balance(amount=ZERO),
```

Comment:
please remember in all your code. this is unnecessary

```suggestion
            balance=Balance(),
```